tutorials/vue/vue-table.md

```diff
@@ -0,0 +1,85 @@
+Vue makes it very easy to use tables in your projects because of its list rendering functionality.
+With a few lines of html and a `v-for` loop, you can have a table that is easy to change should you need
+to make additions or subtractions later down the line. To begin, you must first create the structure of
+the table that you will be designing using the appropriate table tags and labels for your columns and
+rows. Once that is done, you can begin using vue in the `<tbody>` section of the `<table>` tag. For
```

Comment:
Let's get rid of these two sentences: "To begin, you must first create the structure of
the table that you will be designing using the appropriate table tags and labels for your columns and
rows. Once that is done, you can begin using vue in the `<tbody>` section of the `<table>` tag."